server/research/endpoints.js

```diff
@@ -1,22 +1,20 @@
 // For getting all interactions
 function interactionsEndpoint(pool, req, res) {
-  // query!!!
-  const sql = `
-    SELECT
-      id,
-      interaction,
-      timestampz,
-      session
-        
-    FROM interactions`;
+  const sql = `SELECT * FROM interactions`;
 
+  pool.query(sql)
+    .then(results => {
+      console.log(results.rows);
+      console.log(JSON.parse(JSON.stringify(results.rows)));
+      console.log('response ended');
```

Comment:
These are good helpful comments for development, but let's update them so that 1) they are meaningful in log files in production and 2) they don't log user data from the database (but metadata like the total number of rows is fine).

Reply:
hmmmm. so I dont actually think we need these logs in production. Deleting them!